datafusion/proto/proto/datafusion.proto

```diff
@@ -389,7 +389,6 @@ message LogicalExprNode {
     NegativeNode negative = 13;
     InListNode in_list = 14;
     Wildcard wildcard = 15;
-    ScalarFunctionNode scalar_function = 16;
```

Comment:
I think we should add comment to occupy this position of `16` @dmitrybugakov @alamb @Omega359